.github/workflows/on-osx-latest.yml

```diff
@@ -0,0 +1,32 @@
+name: on-osx-latest
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+#      - name: Install GitHub CLI
+#        run: |
+#          brew update
+#          brew install gh
+      - name: compile oommf
+        run: |
+          cat oommf-version
+          pwd
+          ls -l
+          make build
+      - name: run version and platform
+        run: |
+          pwd
+          tclsh oommf/oommf.tcl +version
+          tclsh oommf/oommf.tcl +platform
```

Comment:
The last run did report `OOMMF threads = 3`. Information about the CPU count of the VM running the job might potentially be interesting - if we are interested in (manually) checking as much details as possible.

Reply:
Will add this.